src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.*;
+import duke.dukeexceptions.DeadlineException;
+import duke.dukeexceptions.DukeException;
+import duke.task.Deadline;
+import duke.task.Task;
+
+import java.io.IOException;
+
+public class DeadlineCommand extends Command {
```

Comment:
Every public class should have a comment even if it inherits from abstract class, i recommend writing 😄